Read-through of Therapeutics Text (#965)
greenelab/covid19-review: content/20.pharmaceuticals.md

For example, SARS-CoV-1 was controlled within 9 months of its appearance, whereas SARS-CoV-2 remains a significant global threat almost 1.5 years later.
]({{csse_deaths_figure}} "Global COVID-19 deaths"){#fig:csse-deaths secno=1}

> SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat initially closely paralleled the emergence of _Severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1), which was responsible for the 2002 SARS epidemic.

Reviewer: ```suggestion
SARS-CoV-2's rapid shift from an unknown virus to a significant worldwide threat initially closely paralleled the emergence of _severe acute respiratory syndrome-related coronavirus_ (SARS-CoV-1), which was responsible for the 2002 SARS epidemic.
```

Author: This is their formatting thing, they want the first word capitalized! I think, at least.

Reviewer: Sorry you totally pointed this out before haha

While SARS-CoV-1 belonged to a distinct lineage from the two other HCoVs known at the time of its discovery [@doi:10.1093/ajcp/aqaa029], SARS-CoV-2 is closely related to SARS-CoV-1 and is a more distant relative of another HCoV characterized in 2012, _Middle East respiratory syndrome-related coronavirus_ [@doi:10/ggjr43; @doi:10.1056/NEJMoa1211721].
During the 21st century, significant efforts has been devoted to characterizing these two devastating HCoVs in order to better understand how they interact with human hosts.
Therefore, SARS-CoV-2 emerged under very different circumstances than SARS-CoV-1 in terms of scientific knowledge about HCoVs and the tools available to characterize them.

Despite the apparent advantages for responding to SARS-CoV-2 infections, the trajectory of COVID-19 has led to a much larger number of deaths than that of SARS (Figure @fig:csse-deaths).
By July 2003, the SARS outbreak was officially determined to be under control, with the success credited to infection management practices such as mask wearing [@doi:10.1038/nrmicro.2016.81].

> MERS-CoV is still circulating and remains a concern; although the fatality rate is very high at almost 35%, the disease is much less easily transmitted, as its R~0~ has been estimated to be 1 [@doi:10.1093/ajcp/aqaa029].

Author: ```suggestion
_Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) is still circulating and remains a concern; although the fatality rate is very high at almost 35%, the disease is much less easily transmitted, as its R~0~ has been estimated to be 1 [@doi:10.1093/ajcp/aqaa029].
```

Small molecule drugs today encompass a wide range of therapeutics beyond antibiotics, including antivirals, protein inhibitors, and many broad-spectrum pharmaceuticals.
Broad-spectrum treatments are not specifically targeted at a virus or at particular host receptors, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
Instead, when a virus enters a host, the host becomes the virus's environment, and, therefore, the state of the host can also influence the virus's ability to replicate and spread.
The administration and assessment of such treatments on a rapid time course was feasible because they are often either available in hospitals, or in some cases may also be prescribed to a large number of out-patients.
One of the other advantages is that these well-established compounds, if found to be beneficial, tend to be more widely available than boutique experimental drugs.

In some cases, prior data was available from experiments examining the response of other HCoV or HCoV infections to a candidate drug.

> In addition to non-pharmaceutical interventions such as encouraging non-intubated patients to adopt a prone position [@doi:10.1001/jamainternmed.2020.3030], knowledge about interactions between HCoV and the human body, many of which emerged from SARS and MERS research over the past two decades, led to the suggestion that a number of common drugs might provide benefits for COVID-19 patients.

Reviewer: ```suggestion
In addition to non-pharmaceutical interventions such as encouraging non-intubated patients to adopt a prone position [@doi:10.1001/jamainternmed.2020.3030], knowledge about interactions between HCoV and the human body, many of which emerged from SARS and MERS research over the past two decades, led to the suggestion that a number of common drugs might benefit COVID-19 patients.
```

The three HCoV that have emerged in the past twenty years have presented serious threats, and therefore, clinical and pharmaceutical advancements in treating them have been of significant interest.

> Prior analyses of the virological and pathogenic properties of SARS-CoV-1 and _Middle East respiratory syndrome-related coronavirus_ (MERS-CoV) have provided a strong foundation for the development of hypotheses about SARS-CoV-2 that offered insight into potential therapeutic approaches.

Reviewer: ```suggestion
Prior analyses of the virological and pathogenic properties of SARS-CoV-1 and MERS-CoV have provided a strong foundation for the development of hypotheses about SARS-CoV-2 that offered insight into potential therapeutic approaches.
```

In fact, corticosteroids such as dexamethasone are sometimes used to treat CRS [@doi:10.1182/blood-2014-05-552729].
Guidelines were quickly updated to encourage the use of dexamethasone in severe cases [@doi:10.1001/jama.2020.16747], and this affordable and widely availalable treatment rapidly became a valuable tool against COVID-19, with demand surging within days of the preprint's release [@doi:10.1136/bmj.m2512].
In the months since its release, dexamethasone has become a valuable tool in the treatment of severe COVID-19 [@doi:10.1016/j.ejphar.2021.173854].

#### Treatments Adapted from Clinical Practice

Some of the most widely publicized examples of efforts to repurpose drugs for COVID-19 are broad-spectrum, small-molecule drugs where the mechanism of action made it seem that the drug might target SARS-CoV-2.
However, the exact outcomes of such treatments are difficult to predict _a priori_, and there are several examples where early enthusiasm was not borne out in subsequent trials.

> One of the most famous examples analyses of whether a well-known medication could provide benefits to COVID-19 patients came from the assessment of chloroquine (CQ) and hydroxychloroquine (HCQ), which are used for the treatment and prophylaxis of malaria as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@dailymed-hcq].

Reviewer: ```suggestion
One of the most famous examples of an analyses of whether a well-known medication could provide benefits to COVID-19 patients came from the assessment of chloroquine (CQ) and hydroxychloroquine (HCQ), which are used for the treatment and prophylaxis of malaria as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@dailymed-hcq].
```

_In vitro_ analyses reported that CQ inhibited cell entry of SARS-CoV-1 [@doi:10.1186/1743-422X-2-69] and that both CQ and HCQ inhibited viral replication within cultured cells [@doi:10.1093/cid/ciaa237], leading to early hope that it might provide similar therapeutic or protective effects in patients.
However, while the first publication on the clinical application of these compounds to the inpatient treatment of COVID-19 was very positive [@doi:10.1016/j.ijantimicag.2020.105949], it was quickly discredited [@url:https://www.isac.world/news-and-publications/official-isac-statement].
Over the following months, extensive evidence emerged demonstrating that CQ and HCQ offered no benefits for COVID-19 patients and, in fact, carried the risk of dangerous side effects (Appendix A).
The nail in the coffin came when findings from the large-scale RECOVERY trial were released on October 8, 2020.

> This study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate and used a randomized, open-label design to study the effects of HCQ compared to SOC at 176 hospitals in the United Kingdom [@doi:10.1056/NEJMoa2022926].

Reviewer: ```suggestion
This study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate and used a randomized, open-label design to study the effects of HCQ compared to standard of care (SOC) at 176 hospitals in the United Kingdom [@doi:10.1056/NEJMoa2022926].
```

A meta-analysis that evaluated the results of the RECOVERY trial alongside trials of other corticosteroids, such as hydrocortisone, similarly concluded that corticosteroids may be beneficial to patients with severe COVID-19 who are receiving oxygen supplementation [@doi:10.1053/j.jvca.2020.11.057].
Thus, it seems likely that dexamethasone is useful for treating inflammation associated with immunopathy or cytokine release syndrome (CRS), which is a condition caused by detrimental overactivation of the immune system [@individual-pathogenesis].
In fact, corticosteroids such as dexamethasone are sometimes used to treat CRS [@doi:10.1182/blood-2014-05-552729].

> Guidelines were quickly updated to encourage the use of dexamethasone in severe cases [@doi:10.1001/jama.2020.16747], and this affordable and widely availalable treatment rapidly became a valuable tool against COVID-19, with demand surging within days of the preprint's release [@doi:10.1136/bmj.m2512].

Author: ```suggestion
Guidelines were quickly updated to encourage the use of dexamethasone in severe cases [@doi:10.1001/jama.2020.16747], and this affordable and widely available treatment rapidly became a valuable tool against COVID-19, with demand surging within days of the preprint's release [@doi:10.1136/bmj.m2512].
```

In this case, rather than introducing ARBs/ACEIs, a number of analyses have investigated whether discontinuing use affects COVID-19 outcomes.
An initial observational study of the association of exposure to ACEIs or ARBs with outcomes in COVID-19 was retracted from the _New England Journal of Medicine_ [@doi:10.1056/NEJMoa2007621] due to concerns related to data availability [@doi:10.1056/nejmc2021225].
As randomized trials have become available, they have demonstrated no effect of continuing versus discontinuing ARBs/ACEIs on patient outcomes [@doi:10/fvgt; @doi:10.1001/jama.2020.25864] (Appendix A).
Thus, once again, despite a potential mechanistic association with the pathology of SARS-CoV-2 infection, these medications were not found to influence the trajectory of COVID-19 illness.
However, because the medications were so widespread, clinical research was able to evaluate their efficacy very quickly.

> This feasbility presents a double-edged sword, because HCQ/CQ were incorporated into SOC in many countries before being removed once the evidence indicated its potential to harm COVID-19 patients [@url:https://www.fda.gov/media/138946/download; @url:https://www.ema.europa.eu/en/news/covid-19-chloroquine-hydroxychloroquine-only-be-used-clinical-trials-emergency-use-programmes].

Author: ```suggestion
This feasibility presents a double-edged sword, because HCQ/CQ were incorporated into SOC in many countries before being removed once the evidence indicated its potential to harm COVID-19 patients [@url:https://www.fda.gov/media/138946/download; @url:https://www.ema.europa.eu/en/news/covid-19-chloroquine-hydroxychloroquine-only-be-used-clinical-trials-emergency-use-programmes].
```
